pkg/users/local.js

```diff
@@ -752,8 +752,8 @@ PageAccount.prototype = {
 
            if (data[data.length - 1].indexOf('**Never logged in**') > -1)
                return null;
-           else
-               return new Date(data[data.length - 1]);
+            return moment(data[data.length - 1]).isValid() ?
+                moment(data[data.length - 1]).format('LLL') : data[data.length - 1];
```

Comment:
This still doesn't work right:
 - Click on "never lock", select "lock account on", and select a date (I tried June 30
 - The dialog now shows the localized date "30.06.2018"; press "Change"
 - The link now says "Lock account on Jun 30, 2018" (not localized)
 - Click on the link again, the dialog is now pre-filled with "2018-06-30"; clicking on "change" says "invalid date"

Again, if this is too hairly, let's leave out the datepicker here. If you do fix it, please squash the commits together.